demo/build.gradle

```diff
@@ -26,10 +27,18 @@ dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation 'com.android.support:multidex:1.0.3'
     implementation project(':fullstory-segment-middleware')
 
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
 
+}
+
+apply plugin: 'fullstory'
+fullstory {
+    org '56EM'
+    server 'https://staging.fullstory.com'
```

Comment:
is there a way to commit something that doesn't have an actual org and server value? Can you use environment variables?